devel/create_distro_tarball.sh

```diff
@@ -65,163 +66,130 @@ src/build/gyp_chromium -D use_system_libs=1 \$*
+    $DIR/build \
+    --exclude $DIR/build/android/arm-linux-androideabi-gold \
+    $DIR/install \
+    $DIR/net/instaweb \
+    $DIR/pagespeed \
+    $DIR/strings \
     $GTEST \
-    $DIR/src/third_party/apr/apr.gyp \
-    $DIR/src/third_party/aprutil/aprutil.gyp \
-    $DIR/src/third_party/aprutil/apr_memcache2.h \
-    $DIR/src/third_party/aprutil/apr_memcache2.c \
-    $DIR/src/third_party/httpd/httpd.gyp \
-    $DIR/src/third_party/httpd24/httpd24.gyp \
-    $DIR/src/third_party/base64 \
-    $DIR/src/third_party/brotli \
-    $DIR/src/third_party/chromium/src/base \
+    $DIR/third_party/apr/apr.gyp \
+    $DIR/third_party/aprutil/aprutil.gyp \
+    $DIR/third_party/aprutil/apr_memcache2.h \
+    $DIR/third_party/aprutil/apr_memcache2.c \
+    $DIR/third_party/httpd/httpd.gyp \
+    $DIR/third_party/httpd24/httpd24.gyp \
+    $DIR/third_party/base64 \
+    $DIR/third_party/brotli \
+    $DIR/third_party/chromium/src/base \
     --exclude src/third_party/chromium/src/base/third_party/xdg_mime \
     --exclude src/third_party/chromium/src/base/third_party/xdg_user_dirs \
-    $DIR/src/third_party/chromium/src/build \
-    --exclude $DIR/src/third_party/chromium/src/build/android \
-    $DIR/src/third_party/chromium/src/googleurl \
-    $DIR/src/third_party/chromium/src/net/tools \
-    $DIR/src/third_party/closure/ \
-    $DIR/src/third_party/closure_library/ \
-    $DIR/src/third_party/css_parser \
-    $DIR/src/third_party/domain_registry_provider \
+    $DIR/third_party/chromium/src/build \
+    --exclude $DIR/third_party/chromium/src/build/android \
+    $DIR/third_party/chromium/src/googleurl \
+    $DIR/third_party/chromium/src/net/tools \
+    $DIR/third_party/closure/ \
+    $DIR/third_party/closure_library/ \
+    $DIR/third_party/css_parser \
+    $DIR/third_party/domain_registry_provider \
     $GFLAGS \
     $GIFLIB \
-    $DIR/src/third_party/google-sparsehash \
-    $DIR/src/third_party/grpc \
-    $DIR/src/third_party/hiredis \
+    $DIR/third_party/google-sparsehash \
+    $DIR/third_party/grpc \
+    $DIR/third_party/hiredis \
     $ICU \
     $JSONCPP \
-    $DIR/src/third_party/libjpeg_turbo/libjpeg_turbo.gyp \
-    $DIR/src/third_party/libpng/libpng.gyp \
+    $DIR/third_party/libjpeg_turbo/libjpeg_turbo.gyp \
+    $DIR/third_party/libpng/libpng.gyp \
     $LIBWEBP \
-    $DIR/src/third_party/modp_b64 \
-    $DIR/src/third_party/optipng \
+    $DIR/third_party/modp_b64 \
+    $DIR/third_party/optipng \
     $PROTOBUF  \
-    $DIR/src/third_party/rdestl \
+    $DIR/third_party/rdestl \
     $RE2 \
-    $DIR/src/third_party/redis-crc \
-    $DIR/src/third_party/serf \
-    $DIR/src/third_party/zlib/zlib.gyp \
-    $DIR/src/tools/gyp \
-    $DIR/src/url
+    $DIR/third_party/redis-crc \
+    $DIR/third_party/serf \
+    $DIR/third_party/zlib/zlib.gyp \
+    $DIR/tools/gyp \
+    $DIR/url
 
-if [ x != x$CHECKOUT_DIR ]; then
-  echo "You may want to rm -rf $CHECKOUT_DIR to clean up"
-fi
+cd "$MPS_CHECKOUT"
+rm -r "$TEMP_DIR"
```

Comment:
Yes, it looks reasonable.  The main change is that `.mk` and `.pyc` files aren't included anymore, which I think was one of your changes?